### DISCUSSION TITLE
SSD1306 128x32 or 128x64 OLED displays

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#326] [Announcements]
![image](https://user-images.githubusercontent.com/2480569/183244459-15601fde-1e42-4e2f-8208-13a7a69071bc.png)
![image](https://user-images.githubusercontent.com/2480569/183244462-5aa703a7-e1ca-48e0-950f-6738bf38b7bc.png)
Thanks to the great work of @diyelectromusic, there is now support for SSD1306 128x32 or 128x64 OLED displays.
Thank you Kevin! :+1:

**Comment by DutchBigEd** [comment]
I'm using the SSD1306 128x64 OLED display in my setup and it would be very beneficial for me if the bank number/name would be displayed on the bottom line of the display.

**Reply by probonopd** [reply]
That's a good idea.
@diyelectromusic do you think it would be very hard to make use of the 2 additional lines on that type of display?

**Reply by diyelectromusic** [reply]
I don't know... maybe, but it would then mean the UI is different for the different types of display which isn't ideal I guess?  I don't really have a strong opinion one way or the other :)

**Reply by probonopd** [reply]
Yes, right now we show the same information on all kinds of supported displays.
Not sure whether deviating from this is a good idea, it would further increase complexity.

**Comment by shawnpinciara** [comment]
Hello, I'm using the SSD1306 128x64 with the options commented but it really doesn't work, changed the oled and retried (also by making sure the current is fine and the address too) but still doesn't work.. how should the minidexed.ini look like?
should the LCDEnabled be =1 or =0?

**Reply by diyelectromusic** [reply]
LCDEnabled should be 1 and SSD1306LCDI2CAddress=0x3C typically but it will depend on your display.

Do you see anything on the HDMI console log?

Kevin

**Reply by shawnpinciara** [reply]
Thanks again here too, it works!

**Comment by johndoe201274** [comment]
Hi All! i'm having problems with my minidexed setup. i'm using a ssd 1306 (0,9 inch model) but i cant get it to work.
Can somebody point me in the right direction. I think my .ini file is correct, but still doesnt work. I used the HD44780 LCD before and this works fine, but de ssd1306 display doesn't. Am i missing something.

Here is the piece of code is my minidexed.ini
# SSD1306 LCD
# For a 128x32 display, set LCDColumns=20; LCDRows=2
# For a 128x64 display, set LCDColumns=20; LCDRows=4
LCDEnabled=1
SSD1306LCDI2CAddress=0x3C
SSD1306LCDWidth=128
SSD1306LCDHeight=32
SSD1306LCDRotate=0
SSD1306LCDMirror=0
LCDColumns=20
LCDRows=2

Help is appreciated.

**Reply by Banana71** [reply]
Hi John,

I don't know if it helps but did you delete the address from the old HD44780 LCD?

```
# HD44780 LCD
LCDI2CAddress=0x00
```
Your settings look fine otherwise

Peter

**Reply by johndoe201274** [reply]
Hi Peter, Thx...do you think it matters? i tried 3 screens and they all have the same behaviour.
im gonna try it later on .... and present you  the outcome. thx again for your quick reply.
